bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/ProperMotion.scala

```diff
@@ -78,13 +75,17 @@ object ProperMotion {
    */
   def properMotionCalculator(baseCoordinates: Coordinates,
                              daysPerYear: Double,
-                             properVelocity: Offset,
+                             properVelocity: Vec2,
                              radialVelocity: Double,
                              parallax: Double,
                              elapsedYears: Double): Coordinates = {
     // We want the baseCoordinates in radians.
-    val (ra,  dec)  = (baseCoordinates.ra.toAngle.toSignedDegrees.toRadians, baseCoordinates.dec.toAngle.toSignedDegrees.toRadians)
-    val (dRa, dDec) = (properVelocity.p.toAngle.toSignedDegrees.toRadians,   properVelocity.q.toAngle.toSignedDegrees.toRadians)
+    val (ra,  dec)  = (baseCoordinates.ra.toAngle.toRadians, baseCoordinates.dec.toAngle.toRadians)
+    // at dec 90 this gets undetermined, while we never observe at dec 90 it wouldn't be nice to have
+    // a division by zero error. At dec 90 we don't correct by cos(dec)
+    val (dRa, dDec) = 
+      if (cos(dec) == 0.0) (properVelocity._1.toRadians, properVelocity._2.toRadians) 
```

Comment:
Looking below, `pos` is `(0,0,1)` at 90º.  The `dRa` is used to calculate the `dPos2` vector which in this case multiplies `dRa` by 0.  I don't know what the right answer is at zero but effectively it seems we could put anything here as `dRa` in this case.

Reply:
Right, we could revisit to try matching what astropy gives you